coderd/devtunnel/servers.go

```diff
@@ -0,0 +1,95 @@
+package devtunnel
+
+import (
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/go-ping/ping"
+	"golang.org/x/exp/slices"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/coder/coder/cryptorand"
+)
+
+type TunnelRegion struct {
+	ID           int
+	LocationName string
+	Nodes        []TunnelNode
+}
+
+type TunnelNode struct {
+	ID                int    `json:"id"`
+	HostnameHTTPS     string `json:"hostname_https"`
+	HostnameWireguard string `json:"hostname_wireguard"`
+	WireguardPort     uint16 `json:"wireguard_port"`
+
+	AvgLatency time.Duration `json:"avg_latency"`
+}
+
+var TunnelRegions = []TunnelRegion{
+	{
+		ID:           1,
+		LocationName: "US East Pittsburgh",
+		Nodes: []TunnelNode{
+			{
+				ID:                1,
+				HostnameHTTPS:     "pit-1.try.coder.app",
+				HostnameWireguard: "pit-1.try.coder.app",
+				WireguardPort:     55551,
+			},
+		},
+	},
+}
+
+func PickTunnelNode() (TunnelNode, error) {
```

Comment:
I might include the picking strategy when connecting to a node. `FindClosestNode` or something to that effect.